src/plugin.rs

```diff
@@ -160,7 +211,9 @@ async fn select(
     values: Vec<JsonValue>,
 ) -> Result<Vec<HashMap<String, JsonValue>>> {
     let mut instances = db_instances.0.lock().await;
-    let db = instances.get_mut(&db).ok_or(Error::DatabaseNotLoaded(db))?;
+    let db = instances
+        .get_mut(&db)
+        .ok_or_else(|| Error::DatabaseNotLoaded(db.to_string()))?;
```

Comment:
`.to_string()` needs to be removed too (otherwise `ok_or_else` would've been correct :P ).
Also, after that, run cargo clippy one last time and we should be done (it will complain about &app references).